logrotate/Options.cmake

```diff
@@ -9,27 +9,50 @@
 #
 # example: cmake -DBUILD_STATIC=ON .. 
 option (BUILD_STATIC "Building g3logrotate as static library. Linking with static g3log library" OFF)
-IF (BUILD_STATIC) 
+IF (BUILD_STATIC)
   SET(G3LOG_LIBRARY_TYPE g3logger)
-  SET(G3LOG_LIBRARY_NAME libg3logger.a)
   MESSAGE("'OPTION: cmake -DBUILD_STATIC=ON'\nLinking statically with g3log. Creating static library for g3logrotate")
+  
+  IF (MSVC)
+    SET(G3LOG_LIBRARY_NAME_DEBUG g3logger_d.lib)
+	SET(G3LOG_LIBRARY_NAME_RELEASE g3logger.lib)
```

Comment:
I am curious about your reasoning for this. 

Why would you need two paths for the g3sinks library? Either you are compiling it for debug or you are compiling it for release, right? You will never compile it for both release and debug in the same go. 

What do you gain for adding ```G3_LIBRARY_PATH_RELEASE``` instead ```G3_LIBRARY_PATH``` and using the standard Cmake way of defining ```debug``` or ```release``` builds?